coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -33,3 +38,83 @@ def __init__(self, documentation, language, docstyle,
 
     def __str__(self):
         return self.documentation
+
+    def parse(self):
+        """
+        Parses documentation independent of language and docstyle.
+
+        :return:
+            A list of the parsed documentation metadata.
+        :raises NotImplementedError:
+            When no parsing method is present for the given language and
+            docstyle.
+        """
+        if self.language == "python" and self.docstyle == "default":
+            return self._parse_documentation_with_symbols((":param ", ": "),
+                                                          ":return:")
```

Comment:
I would suggest you do
```
return self._parse_documentation_with_symbols(
    (":param ", ": "), ":return:")
```
btw not using `:return: `? (so adding a trailing whitespace?)